thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/health/DetectionTaskStatus.java

```diff
@@ -25,6 +25,7 @@
 import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
+import javafx.concurrent.Task;
```

Comment:
I don't think that this package is available for higher jdk versions! This might create an issue!

Reply:
I don't think that this package is available for higher jdk versions! This might create an issue!

Reply:
Thanks for the comment. This import got sneaked in by the IDE. Will remove in following PR.